crates/cheatcodes/src/impls/evm/fork.rs

```diff
@@ -0,0 +1,299 @@
+use alloy_primitives::B256;
+use alloy_sol_types::SolValue;
+
+use super::{Cheatcode, CheatsCtxt, DatabaseExt, Result};
+use crate::{impls::CreateFork, Cheatcodes, Vm::*};
+
+impl Cheatcode for activeForkCall {
```

Comment:
Should these be capitalized in alloy codegen?

Reply:
I don't think it should modify names by default, and instead have a `rename_all` like Serde (not implemented yet)

Reply:
I remember that implementing this in abigen was also a bit painful to get right